### PR TITLE
Added override analytics template chapter

### LINKED ISSUE #189: How to overwrite the google analytics template

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR adds the info how to override the analytics templates.

fixes #189

### CODE REVIEW

File: cookbook/override-analytics-template.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+You are able to override the analytics template with the
+`symfony template overriding mechanism <http://symfony.com/doc/current/book/templating.html#overriding-bundle-templates>`_.
+
+There are three relevant templates:
+
+* ```SuluWebsiteBundle:Analytics/type/google.html.twig```
+* ```SuluWebsiteBundle:Analytics/type/piwik.html.twig```
+* ```SuluWebsiteBundle:Analytics/type/custom.html.twig```
+
+You can access all the information in the twig variable ```analytics```.
```
Comment (danrot) [inline cookbook/override-analytics-template.rst]: Maybe we should explain the options of this variable in a bit more detail?

**Comment by danrot** [comment]
The document needs to be included in a tree.

**Comment by wachterjohannes** [comment]
@danrot fixed

**Comment by danrot** [comment]
@wachterjohannes Tests failing.

**Comment by wachterjohannes** [comment]
@danrot i have only added a missing space between ``blabla`` and the next word ... locally this was no problem but travis dont like that (:

**Pull request merged**